hydrus/app_factory.py

```diff
@@ -38,5 +38,9 @@ def app_factory(api_name: str = "api") -> Flask:
         "/{}/<string:path>/add/<int_list>".format(api_name),
         "/{}/<string:path>/add".format(api_name),
         "/{}/<string:path>/delete/<int_list>".format(api_name))
+    api.add_resource(
```

Comment:
This is not part of the API. I'm not sure if we should serve this here. Shouldn't the diff and all sync related mechanism be served using socket events? Why are we serving this information at the API endpoint. Ideally the app should contain the API and socket events should be done by using `@socket.on` decorators.